tests/clib/test_functions.h

```diff
@@ -142,10 +142,14 @@ double retrieve_DBL_MIN(void);
 double retrieve_DBL_MAX(void);
 void add_complexd(double complex *, double complex *, double complex *);
 void mul_complexd(double complex *, double complex *, double complex *);
+void add_complexld(long double complex *, long double complex *, long double complex *);
```

Comment:
It would be great to have a couple of simple tests for passing `long double` here alongside the `long double complex` tests.